ui/src/components/SampleView/SampleControls.js

```diff
@@ -181,226 +182,219 @@ export default class SampleControls extends React.Component {
-            {process.env.REACT_APP_FOCUSCONTROLONCANVAS
-              ? (
-                <li>
-                  <OverlayTrigger
-                    trigger="click"
-                    rootClose
-                    placement="bottom"
-                    overlay={(
-                      <span className="slider-overlay" style={{ marginTop: '20px' }}>
-                        <OneAxisTranslationControl
-                          save={this.props.sendSetAttribute}
-                          value={focus_motor.value}
-                          min={focus_motor.limits[0]}
-                          max={focus_motor.limits[1]}
-                          step={this.props.steps.focusStep}
-                          motorName={foucs_motor_uiprop.attribute}
-                          suffix={foucs_motor_uiprop.suffix}
-                          decimalPoints={foucs_motor_uiprop.precision}
-                          state={focus_motor.state}
-                          disabled={this.props.motorsDisabled}
-                        />
-                      </span>
-                    )}
-                  >
-                    <Button
-                      name="focus"
-                      type="button"
-                      data-toggle="tooltip"
-                      title="Focus"
-                      className="fas fa-adjust sample-control"
-                    />
-                  </OverlayTrigger>
-                  <span className="sample-control-label">Focus</span>
-                </li>
-              ) : null}
-            <OverlayTrigger
-              trigger="click"
-              rootClose
-              placement="bottom"
-              overlay={(
-                <span className="slider-overlay">
-                  {zoom_motor.limits[0]}
-                  <input
-                    style={{ top: '20px' }}
-                    className="bar"
-                    type="range"
-                    id="zoom-control"
-                    min={zoom_motor.limits[0]}
-                    max={zoom_motor.limits[1]}
-                    step="1"
-                    value={zoom_motor.commands.indexOf(zoom_motor.value)}
-                    disabled={zoom_motor.state !== MOTOR_STATE.READY}
-                    onMouseUp={this.setZoom}
-                    onChange={(e) => {
-                      this.props.setBeamlineAttribute('diffractometer.zoom', zoom_motor.commands[e.target.value])
-                    }}
-                    list="volsettings"
-                    name="zoomSlider"
-                  />
-                  {zoom_motor.limits[1]}
-                </span>
-              )}
-            >
-              <li>
-                <Button
-                  type="button"
-                  data-toggle="tooltip"
-                  title="Zoom in/out"
-                  className="fas fa-search sample-control"
-                  name="zoomOut"
+      <div className={styles.controls}>
+        <Button
+          as="a"
```

Comment:
It was not obvious that this button was a link, so I made it explicit.

Reply:
:)